test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -1111,6 +1112,10 @@ public static String randomDateFormatterPattern() {
         return randomFrom(FormatNames.values()).getName();
     }
 
+    public static GeoPoint randomGeoPoint() {
+        return new GeoPoint(randomDoubleBetween(-90, 90, true), randomDoubleBetween(-180, 180, true));
+    }
+
```

Comment:
Any addition to ESTestCase needs a fair bit of though. I imagine this is fine, but it certainly want's javadoc because this is imported by *everyone* all the time.

Reply:
When I added this I also noted that the line of code added was duplicated all over the place in many other tests, and perhaps adding this function to ESTestCase was a nice cleanup independent of anything in ESQL. Perhaps I should create a separate PR for that, and get it merged into ES?